Map.py

```diff
@@ -12,7 +12,7 @@
 import ee_plugin.utils
 
 
-def addLayer(image: ee.Image, visParams=None, name=None, shown=True, opacity=1.0):
+def addLayer(eeObject, visParams={}, name=None, shown=True, opacity=1.0):
```

Comment:
see the warning in the [docs](https://docs.python.org/3/tutorial/controlflow.html#default-argument-values).
In the function arguments it is better to use `..., visParams=None, ...` and then in the function
```
if visParams is None:
        visParams = {}
```